org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/settings/XMLValidationSettings.java

```diff
@@ -12,6 +12,8 @@
  */
 package org.eclipse.lemminx.extensions.contentmodel.settings;
 
+import java.lang.reflect.Field;
```

Comment:
nope

Reply:
still no

Reply:
I don't know of any other way to test for the regression where we add a new field but don't manage it in the merge function. Do you have any ideas?

Reply:
It's fine in the test, useless in XMLValidationSettings.java

Reply:
Okay. Thank you, missed that